itest/tapd_harness.go

```diff
@@ -47,26 +47,33 @@ var (
 			"allow the postgres fixture to run in total. Needs "+
 			"to be increased for long-running tests.")
 
-	// defaultBackoffConfig is the default backoff config we'll use for
-	// sending proofs.
-	defaultBackoffConfig = proof.BackoffCfg{
-		BackoffResetWait: time.Second,
-		NumTries:         5,
-		InitialBackoff:   300 * time.Millisecond,
-		MaxBackoff:       600 * time.Millisecond,
+	// defaultHashmailBackoffConfig is the default backoff config we'll use
+	// for sending proofs with the hashmail courier.
+	defaultHashmailBackoffConfig = proof.BackoffCfg{
+		BackoffResetWait:    time.Second,
+		NumTries:            5,
+		InitialBackoff:      300 * time.Millisecond,
+		MaxBackoff:          600 * time.Millisecond,
+		InitialReceiveDelay: 200 * time.Millisecond,
 	}
 
-	// defaultProofRetrievalDelay is the default delay we'll use for the
-	// custodian to wait from observing a transaction on-chan to retrieving
-	// the proof from the courier.
-	defaultProofRetrievalDelay = 200 * time.Millisecond
+	// defaultUniverseRpcBackoffConfig is the default backoff config we'll
+	// use for sending proofs with the universe RPC courier.
+	defaultUniverseRpcBackoffConfig = proof.BackoffCfg{
+		SkipInitDelay:       true,
+		BackoffResetWait:    time.Second,
+		NumTries:            5,
+		InitialBackoff:      300 * time.Millisecond,
+		MaxBackoff:          600 * time.Millisecond,
+		InitialReceiveDelay: 200 * time.Millisecond,
+	}
 )
 
 const (
 	// defaultProofTransferReceiverAckTimeout is the default itest specific
 	// timeout we'll use for waiting for a receiver to acknowledge a proof
 	// transfer.
-	defaultProofTransferReceiverAckTimeout = 15 * time.Second
+	defaultProofTransferReceiverAckTimeout = 5 * time.Second
```

Comment:
Were there issues going below 5 here?

Reply:
Things became unstable, yes. But didn't want to invest more time as they ran very stable with 5 seconds.